suite-native/config/README.md

```diff
@@ -1,11 +1,3 @@
 # @suite-native/config
 
 This package contains helper functions based on configuration variables for different environments of our React Native mobile application.
-
-## Config variables
```

Comment:
It might be good to add some comment on how to set/change ENV variables somewhere, wdyt?

Reply:
Check the README now, please